data/data-pipeline/data_pipeline/etl/score/schemas.py

```diff
@@ -0,0 +1,135 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class CensusCountyInfo:
+    USPS: str
+    GEOID: str
+    ANSICODE: int
+    NAME: str
+    POP10: int
+    HU10: int
+    ALAND: str
+    AWATER: int
+    ALAND_SQMI: float
+    AWATER_SQMI: float
+    INTPTLAT: float
+    INTPTLONG: float
+
+
+@dataclass
+class StateInfo:
+    fips: str
+    state_name: str
+    state_abbreviation: str
+    region: str
+    division: str
+
+
+@dataclass
+class Score:
```

Comment:
Actually, some of these might be floats.

Reply:
Yeah, so actually this file is not currently in use and can be removed. This was part of an idea that we could start using a local schema to validate. I have an example of using this in another repo that I can share offline. I think actually we could and should use [pandera](https://pandera.readthedocs.io/en/stable/schema_models.html?highlight=inputschema) to create a schema that we can annotate input/output datasets with - a major problem we see is with typing, for instance, which this can ensure for us. Additionally it can put certain sanity checks in place, such as making sure percentiles are between 0-100 etc.

Reply:
Talked sync - we'll look into this in the future but for now remove since we're not using